### ISSUE TITLE
Other scripting languages

### ISSUE THREAD

**Issue by der** [epimorphics/dclib#11]
jexl is reasonable expressive but is not a full general purpose language.

Support escaping to ruby or javascript?

**Comment by kal** [comment]
@der - any preference for scripting language? Should Python/Jython be on the list too?

**Comment by der** [comment]
> Should Python/Jython be on the list too?

Yes, and this days strongly preferred if there's a way of doing it.
Which begs the question as to whether the future of dclib should be replacement by enhanced RDFMapper!

**Comment by kal** [comment]
Agreed! We would need some sort of analysis of existing usages of dclib and the feature gap between dclib and RDFMapper, but that could be a useful bit of work to do at some point.
